simplified-app-shared/src/main/java/org/nypl/simplified/app/LoginDialog.java

```diff
@@ -451,6 +474,23 @@ public void onAccountLoginFailureDeviceActivationError(final String message)
         }
       });
 
+    in_barcode_scan_button.setOnClickListener(
+        new OnClickListener() {
+          @Override
+          public void onClick(
+              final @Nullable View v) {
+
+            // IntentIntegrator exit will fire on Scan or Back and hit the onActivityResult method.
+            IntentIntegrator
+                .forFragment(myFragment)
+                .setPrompt("Scan your library barcode.")
+                .setBeepEnabled(false)
+                //.setDesiredBarcodeFormats(IntentIntegrator.ALL_CODE_TYPES)
+                //.setOrientationLocked(false)
```

Comment:
Parameters you've set look good. Just needs comment cleanup.

Reply:
See c85d987